spec/models/project_spec.rb

```diff
@@ -33,7 +33,7 @@
       expect(subject).to_not be_valid
     end
 
-    it "should not accept invalid Pivotal Tracker URL" do
+    it 'should not accept invalid Pivotal Tracker URL' do
```

Comment:
is the s format now completely invalid? looks like it redirects to the n format - I just wonder if some old projects that still have the s might encounter problems - easy fix I guess for the maintainer I guess